draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/TitleBlockZen.tsx

```diff
@@ -256,39 +265,40 @@ const renderSubtitle = (
   </div>
 )
 
-const renderSectionTitle = (
-  sectionTitle?: string,
+const defaultRenderSectionTitle = (
+  sectionTitle?: React.ReactNode,
   sectionTitleDescription?: string,
   variant?: Variant,
   sectionTitleAutomationId?: string,
   sectionTitleDescriptionAutomationId?: string
 ): JSX.Element => (
-  <div className={styles.sectionTitleContainer}>
-    <div className={styles.sectionTitleInner}>
-      {sectionTitle && (
-        <div className={styles.sectionTitle}>
-          <Heading
-            variant="heading-2"
-            color={isReversed(variant) ? "white" : "dark"}
-            classNameOverride={styles.sectionTitleOverride}
-            data-automation-id={sectionTitleAutomationId}
-          >
-            {sectionTitle}
-          </Heading>
-        </div>
-      )}
-      {sectionTitleDescription && (
-        <div
-          data-automation-id={sectionTitleDescriptionAutomationId}
-          className={classNames(styles.sectionTitleDescription, {
-            [styles.dark]: !isReversed(variant),
-          })}
+  <>
+    {sectionTitle && typeof sectionTitle === "string" && (
+      <div className={styles.sectionTitle}>
+        <Heading
+          variant="heading-2"
+          color={isReversed(variant) ? "white" : "dark"}
+          classNameOverride={styles.sectionTitleOverride}
+          data-automation-id={sectionTitleAutomationId}
         >
-          {sectionTitleDescription}
-        </div>
-      )}
-    </div>
-  </div>
+          {sectionTitle}
+        </Heading>
+      </div>
+    )}
+    {sectionTitle && typeof sectionTitle !== "string" && (
```

Comment:
Just a wondering if this is here to showcase the use of the ReactNode approach over the custom render prop? My confusion just sprouts from React Node not being a type that can be passed to section title (at least for a TS perspective)
![Screenshot 2023-01-26 at 11 14 18 am](https://user-images.githubusercontent.com/36558508/214723312-e3f98f60-8465-49fc-ad62-0d5b5a24052d.png)

Reply:
Could we potentially extend what the `sectionTitle`prop can take and not introduce a new render prop? ie:
```
sectionTitle?:
    | string
    | React.ReactNode
    | ComponentType<SectionTitleRenderProps>
```
Then it could look like this:
sectionTitle={<CustomComponent />}
or this:
sectionTitle={<div >This may be a bit of chaos by won't clamp down on types we assume?</div>}

Curious to hear peeps thoughts 🤔

Reply:
This is a great callout, that `defaultRenderSectionTitle` shouldn't accept a `ReactNode`.. That's an artifact from the previous approach.

I've pushed a fix that leaves the `defaultRenderSectionTitle` exactly the same as before, barring the name, and it leaves out the two styling `div`s, since those are handled in the main title block render.

I like the idea of simply injecting the `sectionTitle` as a ReactNode directly in the previous render function, but conditionally rendering it in a `Heading` in the local render function.  Typing can be a little silly here, since `ReactNode` composes `string`, but I'm sure we can work around that.

Glad to hear any feedback